WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -27,6 +27,10 @@ final class OrderDetailsViewController: UIViewController {
         return EntityListener(storageManager: ServiceLocator.storageManager, readOnlyEntity: viewModel.order)
     }()
 
+    /// Coordinator for shipping label reprint UX navigation.
+    ///
+    private lazy var reprintShippingLabelCoordinator = ReprintShippingLabelCoordinator(sourceViewController: self)
```

Comment:
Wondering if it is possible to have the same lifecycle for Coordinator and view controllers that it coordinates. So we would create a coordinator when we need to start a flow and it would deallocate once all the screens from the flow are gone.